server/hedley/modules/custom/hedley_aggregated_ncda/js/elm.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file
+ *  Elm applications.
+ */
+
+(function ($, Drupal) {
+
+  /**
+   * Adds Elm application.
+   */
+  Drupal.behaviors.ElmApps = {
+    attach: function (context, settings) {
+      var elmApps = settings.elm_apps;
+      // Iterate over the apps.
+      Object.keys(elmApps).forEach(function (appName) {
+        // appName with unique css ID, e.g. `elm-app--2`.
+        var node = document.getElementById(appName);
+        if (!node) {
```

Comment:
is it on purpose that we do not use jQuery for this? Also `context` and `once`?

Reply:
@AronNovak 

Whole Elm app was ported from WFP-INCE.
Checked several other projects where we attach Elm apps, and we have this code at all.
Do you see a problem with current implementation?

Reply:
the only thing is that `attached` have the potential to be called multiple times during the lifetime of a page. Could it be problematic? If not, I do not see any issues.

Reply:
Never seen functionality issue here, so, I don't think this method cause any problems.